Added introduction file

DustinAllan opened the pull request:
Introducing myself - Dustin

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
Added introduction file

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+Hello,
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

DustinAllan committed:
Added favorite code school path to introduction.md, as requested.

DustinAllan committed:
Altered introduction.md, changed my favorite code school path

DustinAllan committed:
Add three deadly skills to introduction.md

DustinAllan committed:
Add "Killing history using git rebase" to my list of deadly skills in introduction.md file

codeschool-kiddo committed:
Added new rule about coffee

DustinAllan committed:
fix errors in dojo_rules.md file

DustinAllan committed:
Merge pull request #2 from DustinAllan/new_rules

Too many people are spilling coffee

DustinAllan committed:
Merge branch 'deadly_skills'

DustinAllan committed:
Edit kill_list.md to contain bad coding practices

DustinAllan committed:
Merge release_branch_1.0 into master branch

DustinAllan committed:
Add my own programmer grievance. fixes #3

DustinAllan committed:
Add my own programmer grievence. fixes #3

DustinAllan committed:
Merge branch 'master' into kill_list

DustinAllan committed:
Merge pull request #4 from DustinAllan/kill_list

Added  own programmer grievance. fixes #3

DustinAllan committed:
Update README.md

Updated README.md to include instructions that all members must read the rules. Also added a link to deadly vipers home page.
